modules/340-monitoring-kubernetes/images/kubelet-eviction-thresholds-exporter/main.go

```diff
@@ -183,7 +186,7 @@ func generateMetrics() error {
 		log.Fatal(err)
```

Comment:
you forgot to replace imageFsInodesAvail -> imageFsInodes in line 184

Reply:
Thank you very much for noticing. I fixed it.